pkg/services/rpcsrv/server.go

```diff
@@ -196,7 +248,7 @@ var invalidBlockHeightError = func(index int, height int) *neorpc.Error {
 var upgrader = websocket.Upgrader{}
 
 // New creates a new Server struct.
-func New(chain blockchainer.Blockchainer, conf config.RPC, coreServer *network.Server,
+func New(chain Ledger, conf config.RPC, coreServer *network.Server,
```

Comment:
We already have a dependency on `core`, do we need this interface at all? It seems all callsites use `core.Blockchain`.

Reply:
I hope we can remove it in the future. That's the model we have for all of the other services and RPC shouldn't be different.